.github/workflows/lint-test.yml

```diff
@@ -131,13 +131,19 @@ jobs:
     - uses: actions/checkout@v3
 
     - name: Set up environment
+      id: setup
       uses: ./.github/actions/setup-env
       with:
         python-version: 3.8
         nox-setup: tools
 
     - name: Run slotscheck
-      run: task slotscheck
+      if: (success() || failure()) && steps.setup.outcome == 'success'
+      run: nox -s slotscheck
+
+    - name: Run check-manifest
+      if: (success() || failure()) && steps.setup.outcome == 'success'
+      run: nox -s check-manifest
 
```

Comment:
perhaps after this step we should run `python -m build` to test that the project properly builds..?

Reply:
Once we have a proper build+release workflow we should use that instead, but for now I've added a build step in https://github.com/DisnakeDev/disnake/pull/674/commits/a57887200392f62b993a705d1622af8057098a84.